script/demoScripts/demoNEARIntents.ts

```diff
@@ -0,0 +1,794 @@
+/// ==== SUCCESS TXs ====
+// Bridge USDC: Arbitrum → Solana
+// src: https://arbiscan.io/tx/0x5dc9f72fc5d61d3023755465d1d68d8917410e67c9dcdbd13b1059eb6341ea56
+// dst: https://solscan.io/tx/44EfSioHRApBAAcjQfGM2SNtXEczNocrqZbxjW5CwCcCjb3okhBeVc8635Jy3n1xsqqRfbgz6UoB9y7uXuyEbNup
+// Swap USDT→USDC and bridge: Arbitrum → Solana
+// src: https://arbiscan.io/tx/0x43378460863db76ee5ad96e93727c3e0c6cefd839f1daa8924a1eaef067f410e
+// dst: https://solscan.io/tx/4oUbFCxHByLg7HbeJwfZMkF36ZRokRZYR21qg6WdYMChPNVaQbw22XnWaxKrn7RtHhgvjukXSNwzpdHWTKR5sDHi
+// Bridge USDC: Arbitrum → Base
+// src: https://arbiscan.io/tx/0x865ad42ab96d2e8e5ad742704c134def6f61982a2c42e5248c661a889389e054
+// dst: https://basescan.org/tx/0xdbebd74aea9defd492ed82c67e24b9d0b5e30dac0328ffbfd41e3174b0bd9797
+
+import { randomBytes } from 'crypto'
+
+import { runMain, defineCommand } from 'citty'
+import { config } from 'dotenv'
+import { BigNumber } from 'ethers'
```

Comment:
is the usage of ethers required here?

Reply:
My comment meant: 
Can we not do this without ethers (as we agreed to not use ethers anymore)? Or is using ethers the only way for this?